pages/1.12/cli/command-reference/dcos-security/dcos-security-cluster/dcos-security-cluster-secret-store-seal-status/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+navigationTitle:  dcos security cluster secret-store seal-status
+title: dcos security cluster secret-store seal-status
+menuWeight: 92
+excerpt: Viewing the seal status of a secrets store
+enterprise: true
+---
+
+# Description
+
+The `dcos security cluster secret-store seal-status` command displays the seal status of a secrets store.
+
+# Usage
+
+```
+dcos security cluster secret-store seal-status [OPTIONS] STORE_ID
+```
+
+# Options
+
+| Name |  Description |
+|---------|-------------|
+| `STORE_ID`  | ID of secret store. (Required)|
```

Comment:
In `dcos auth list-providers` command line arguments of this form are listed under "Positional arguments".

Reply:
Ok, thanks. Here's what I got from the help command:

`Usage: dcos security cluster secret-store seal-status [OPTIONS] STORE_ID`

The options only listed --json and --help, and there are no positional arguments listed in the CLI output. I had to guess. :) I will fix this as noted.